src/pages/studies/index.js

```diff
@@ -1,10 +1,110 @@
 import matter from 'gray-matter';
-import Link from 'next/link';
+// import Link from 'next/link';
 
 export default function StudyIndex(props) {
-  return props.studies.map(s => {
-    return (<Link key={s.slug} href={`/studies/${s.slug}`}>{s.frontmatter.title}</Link>)
-  });
+  // Listing all posts under content/studies
+  // return props.studies.map(s => {
+  //   return (<Link key={s.slug} href={`/studies/${s.slug}`}>{s.frontmatter.title}</Link>)
+  // });
+
+  return (
+    <main className="study-index">
+      <section className="study-index__heading pt5 pb4">
+        <h1 className="text-header-sm text-center">Case Studies</h1>
+      </section>
+
+      <section className="study-index__studies site-padding-x">
+        <div className="study-index-case-study-card case-study-card flex flex-row justify-between mxauto mb1_5 md:mb3">
```

Comment:
Nit: under BEM, this should be `study-index__case-study-card`

Reply:
thanks for catching that! should be fixed now :)